packages/dev_disc/README.md

```diff
@@ -58,33 +58,33 @@ Banana23 = iotcc	# Later user could create Liota Package for discovered devices
 
 [DEVSIM_CFG]
 
-devsim_cmd_msg_pipe = /var/tmp/liota/devsim_cmd_messenger.fifo	# the named pipe path for Device Simulator CmdMessengerThread
+devsim_cmd_msg_pipe = /etc/liota/packages/dev_disc/devsim_cmd_messenger.fifo # named pipe for Device Simulator CmdMessengerThread
 
 [DISC_ENDPOINT_LIST]		# Endpoing list where you want discovery listens on and simulator send messages to
 
 							# if no item in this list, Device Discovery will not be started
 
-disc_msg_pipe = /var/tmp/liota/discovery_messenger.fifo		# currently, only support these 4 types
+disc_msg_pipe = /etc/liota/packages/dev_disc/discovery_messenger.fifo	# currently, support these 4 types (currently,
 
-socket = 127.0.0.1:5000						# you can only use some of them by deleting others
+socket = 127.0.0.1:5000						# coap and socket are not allowed for security consideration).
 
-mqtt = 127.0.0.1:1883:device_discovery				# IP address should be updated according to your system
+mqtt = 127.0.0.1:1023:device_discovery				# IP address should be updated according to your system
```

Comment:
that is old comments for socket and coap. Will delete that line for current codes.